### PR TITLE
✨Add Join button component

### PR DESCRIPTION
**Pull request by fre-ben**
- Add Join button
- Add style to button
- Add button component to storybook

Deployment: https://sisu-pipelin-joinbtn-6hb7ahfeb.herokuapp.com/storybook/index.html?path=/story/common-button--join
Part of issue #1

### CODE REVIEW

File: components/button/Button.module.css [hunk 1]
```diff
@@ -44,3 +44,17 @@
...
+  --bottom-border-length: 62%;
+
+  font-size: 2.2em;
+  padding: 0.25em 0.22em;
+}
+
+.joinBackground {
+  background-color: var(--primary-color);
+  padding: 0.05em 0.23em;
+  box-shadow: 0 3px 10px #000000c7;
```
Comment (mhstrkmp) [inline components/button/Button.module.css]: Maybe also use a variable for the shadow color if you plan to use it in more components. Then you would only have to modify it in one place if you change your mind.

**Review by mhstrkmp — COMMENTED** [review]
lgtm 🚀

**Review by DNaunin — APPROVED** [review]
Good Job! 🚀

**Comment by SilviaBarcellini** [comment]
lgtm 👍

File: components/button/Button.module.css (hunk 1 above)
Comment (fre-ben) [reply components/button/Button.module.css]: Good idea, will do that!

**Pull request merged**